spec/game_logic_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../lib/game_logic.rb'
+RSpec.describe Game do
+  describe '#display_board' do
+    let(:field) do
```

Comment:
if we use a variable repeatedly in our tests, we can move this variable to the top after line 2 so that all of our tests have access to it. This favors the DRY principle [REQUIRED].